csm/core/blogic/const.py

```diff
@@ -28,6 +28,8 @@
 CSM_CONF_PATH = ETC_PATH + "/csm"
 CORTXCLI_CONF_PATH = ETC_PATH + "/cli"
 CSM_SOURCE_CONF = "{}/conf/etc/csm/csm.conf".format(CSM_PATH)
+CSM_DEFAULT_CONF = "{}/conf/csm_default_config.yaml".format(CSM_PATH)
+CSM_DEFAULT_DB = "{}/conf/csm_default_db_config.yaml".format(CSM_PATH)
```

Comment:
can we rename these files as below,
csm_default.conf
csm_default_db.conf

so that these files will be in sync with existing csm.conf

your thoughts @pawankumarsrivastava ?